src/components/button/button.stories.ts

```diff
@@ -156,3 +156,6 @@ export const darkModeRTL_TestOnly = (): string => html`
 `;
 
 darkModeRTL_TestOnly.parameters = { modes: modesDarkDefault };
+
+export const outlineNeutralBorderColor_TestOnly = (): string =>
+  html`<calcite-button appearance="outline" kind="neutral">Test</calcite-button>`;
```

Comment:
Nitpick: do we need to add kind here? it will be set to default value if not parsed.

Reply:
The issue specifies that specific combination of props that gives a wrong border.